modules/packages/UnitTest.chpl

```diff
@@ -395,23 +395,24 @@ module UnitTest {
         if len1 == len2 {
           if all(seq1 == seq2) then return;
         }
+        var shorterLength = if len1 < len2 then len1 else len2;
```

Comment:
Don't need to change it, but FWIW could just do `shorterLength = min(len1, len2)`